src/Config/Drupal.php

```diff
@@ -43,6 +44,7 @@ public function __construct()
         $this
             ->registerCustomFixers([
                 new BlankLineBeforeEndOfClass($this->getIndent(), $this->getLineEnding()),
+                new BlankLineAfterStartOfClass($this->getIndent(), $this->getLineEnding()),
```

Comment:
To put before `new BlankLineBeforeEndOfClass` to have the same order as in the use statements and in the rule set?